src/java.base/share/classes/java/util/spi/LocaleServiceProvider.java

```diff
@@ -120,13 +120,13 @@
  * property on the java launcher command line. Setting it at runtime with
  * {@link System#setProperty(String, String)} is discouraged and it may not affect
  * the order.
- * <p>
- * Java Runtime Environment provides the following four locale providers:
+ * @implNote Java Runtime Environment provides the following four locale providers:
```

Comment:
The change looks okay and I agree that this section should be changed to be an implNote. In several other areas, the wording is "The JDK Reference Implementation" rather than "Java Runtime Environment" in implNote text so we might have to adjust that.